oci/container.go

```diff
@@ -305,3 +306,8 @@ func (c *Container) SetCreated() {
 func (c *Container) Created() bool {
 	return c.created
 }
+
+// Description returns a description for the container
+func (c *Container) Description() string {
+	return fmt.Sprintf("%s/%s/%s", c.Labels()[types.KubernetesPodNamespaceLabel], c.Labels()[types.KubernetesPodNameLabel], c.Labels()[types.KubernetesContainerNameLabel])
```

Comment:
Is that safe or do we need `path.Join(....)` instead?

Reply:
Well if this is not a path, then it potentially could cause issues.  I guess only difference was on Windows it would pring %s\%s\%s

Reply:
This is only a description string that shows up when one does crio-conmon-id.scope. That makes it easier to debug.